monocle/worker.py

```diff
@@ -780,8 +780,43 @@ def travel_speed(self, point):
                         if not cooldown or time() > cooldown / 1000:
                             spinning = LOOP.create_task(self.spin_pokestop(pokestop))
                 else:
-                    DB_PROC.add(self.normalize_gym(fort))
 
+                    fort['name'] = ""
```

Comment:
why is this needed?

if there is an error that ['name'] then it should be either set in normalize_gym or accessed with gym.get('name')